util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl

```diff
@@ -1,5 +1,5 @@
 Liferay.Util.openToast({
-   displayType: 'success',
+   display: 'success',
```

Comment:
Shouldn't this be `type` as well?

Reply:
mygod too many things in my head, sorry 🤪

Reply:
![chores](https://user-images.githubusercontent.com/905006/88635510-a1c33300-d0af-11ea-943f-39a294618bc4.gif)